services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/outputs/_manager.py

```diff
@@ -149,9 +149,9 @@ def _remove_downloads(future: Future) -> None:
                     + "".join(
                         # pylint:disable = unexpected-keyword-arg, no-value-for-parameter
                         traceback.format_exception(
-                            etype=type(future._exception),
-                            value=future._exception,
-                            tb=future._exception.__traceback__,
+                            type(future._exception),
+                            future._exception,
+                            future._exception.__traceback__,
```

Comment:
why do the keywords not work any longer?
also check the that the above pylint message not to be deprecated

Reply:
- Changed in version 3.10: The etype parameter has been renamed to exc and is now positional-only.
- Since Python 3.10, instead of passing value and tb, an exception object can be passed as the first argument. If value and tb are provided, the first argument is ignored in order to provide backwards compatibility.
- more info: https://docs.python.org/3.10/library/traceback.html

Reply:
I have updated this function everywhere so it just uses one argument an exception object.